app/models/concerns/auction_scopes.rb

```diff
@@ -18,12 +19,12 @@ module AuctionScopes
     scope :in_reverse_chron_order, -> { order('ended_at DESC') }
     scope :pending_acceptance, -> { where(delivery_status: delivery_statuses['pending_acceptance']) }
     scope :pending_delivery, -> { where(delivery_status: delivery_statuses['pending_delivery']) }
-    scope :not_paid, -> { where(paid_at: nil) }
-    scope :paid, -> { where.not(paid_at: nil) }
-    scope :published, -> { where(published: publishes['published']) }
+    scope :not_paid, -> { not_archived.where(paid_at: nil) }
+    scope :paid, -> { not_archived.where.not(paid_at: nil) }
+    # scope :published, -> { not_archived.where(published: publishes['published']) }
```

Comment:
These scopes are actually not used (why I commented them out and will delete) since the scopes are automagically defined elsewhere